rpc/interface/src/lib.rs

```diff
@@ -76,21 +76,12 @@
 	// TODO
 	rustdoc::bare_urls,
 
+    clippy::multiple_crate_versions,
```

Comment:
the indenting is different here, I think the other lints are incorrect though not this line.

Reply:
I guess `rustfmt` doesn't work inside `#![]`, fixed in other crates too.